contracts/crowdsale/validation/IndividuallyCappedCrowdsale.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity ^ 0.4.18;
+
+import "../../math/SafeMath.sol";
+import "../Crowdsale.sol";
+import "../../ownership/Ownable.sol";
+
+/**
+ * @dev Crowdsale with per-user caps
+ */
+contract IndividuallyCappedCrowdsale is Crowdsale, Ownable {
```

Comment:
Wouldn't be more appropriate to say `UserCappedCrowdsale`? see the names of the functions below

Reply:
ok, this was @ajsantander's original proposal, I changed it because this sounded to me like "capped _by_ the user", but we can go back if you all favor this :)